src/dosbox.cpp

```diff
@@ -796,6 +796,20 @@ void DOSBOX_Init()
 	                  "On 'auto' the mode is determined by 'sbtype'.\n"
 	                  "All OPL modes are AdLib-compatible, except for 'cms'.");
 
+	pstring = secprop->Add_string("opl_fadeout", when_idle, "off");
+	pstring->Set_help(
+	        "Fadeout the OPL synth output after the last IO port write:\n"
+	        "  off:       Don't fadeout; residual output will play forever (default).\n"
+	        "  on:        Wait 350 ms before fading out over a 350 ms period.\n"
+	        "  <custom>:  A custom fadeout definition in the following format:\n"
+	        "               WAIT FADE\n"
+	        "             Where WAIT is how long (in milliseconds) before fading begins,\n"
+	        "             and FADE is the duration (in milliseconds) of the fade out period.\n"
+	        "             Both values can range from 100 to 10000 (milliseconds).\n"
```

Comment:
10s seems excessive, especially for fade outs.

I think I'd use 10 ms to 3s for fades (even 3s is very long, but let's give them a generous range, while also allowing almost instant 10ms fades).

Then for wait I don't think anyone would want to listen for a static tone for 10 seconds before fading it out. Maybe 100ms to 5s?

Reply:
If the last values isn't a tone by stuck in a DC offset, a 10 ms ramp down produces a 100 Hz triangle waveform, which is higher pitch than our 20 Hz high-pass filter. (I've heard these digital ramps before; they kind of sound like a soft plastic zippering sound!)

So that's where the 100 ms lower bound came from; but if the user really wants a near-instant cut-off, then lets let them have it :-)

Reply:
Interesting, from memory I've been using short 10-20ms fades all the time in DAW work, but I might misremember 😄 I'll pay attention to the zipper noises next time 😄